v4/export/config.go

```diff
@@ -14,51 +15,51 @@ import (
 )
 
 type Config struct {
-	Databases []string
-	Host      string
-	User      string
-	Port      int
-	Password  string
+	Databases []string `json:"Databases"`
+	Host      string   `json:"Host"`
+	User      string   `json:"User"`
+	Port      int      `json:"Port"`
+	Password  string   `json:"-"` // omit it for privacy
```

Comment:
only this line is necessary. all other lines are redundant.

Reply:
address in cc939a3